pootle/core/mixins/treeitem.py

```diff
@@ -221,7 +221,8 @@ def get_critical_url(self, **kwargs):
 
     def get_stats(self, include_children=True):
         """get stats for self and - optionally - for children"""
-        self.initialize_children()
+        if include_children:
```

Comment:
@phlax, children are necessary here in any case. This is un-cached TreeItem and any `._calc()` method requires children.